### DISCUSSION TITLE
Settings Tab

### DISCUSSION THREAD

**Discussion by barepixels** [MoonRide303/Fooocus-MRE#35] [Ideas]
A1111 has a central settings tab.  I think Fooocus should have one

set save as JPG or PNG
set your favorite styles you want to choose from
set a default style
set your favorite resolutions you want to choose from
set a default resolution
set your models path
set a default model
set default lora
set use refinner or not
set save metada to json or image or both
default negative prompt
and any other settings now and in the future.

All in one central place. Set it and press Save.  Fooocus will write one settings.json.  

Long style and resolution list are counter-productive and not so elegant

Take control of settings now before it becomes a mess

**Comment by MoonRide303** [comment]
@barepixels I see the whole "Advanced" panel on the right as setting, just divided into categories - basic settings, img2img settings, model settings, sampling settings, misc settings.

What I plan to do is giving users possibility to store current settings by simply hitting one button, "Save Settings" - it would generate settings.json from current settings, and automatically load it after restarting the app (as suggested in [vanilla issue #150](https://github.com/lllyasviel/Fooocus/issues/150)).

**Reply by ke1ne** [reply]
@MoonRide303 Dude, please, be careful with user requests 'oh add this nasty feature, it exists in auto11, I want to here too!!".
Don't convert it to another 11111 with spaceship-like conrol panels, amount of buttons and out-of-control bugs.
Thank you!

**Reply by MoonRide303** [reply]
@ke1ne I hear you. I already said it, but will say it again - I want to keep healthy balance. I want to include most useful and commonly desired features (for which I can do simple and self-explanatory UI), and which I might want to use myself, but I have no intention to become ComfyUI or A1111 clone. MRE will sit somewhere between vanilla Fooocus, and those advanced tools.
